webpack.config.js

```diff
@@ -11,7 +12,11 @@ module.exports = {
         filename: 'webchat.js'
     },
     resolve: {
-        extensions: ['.ts', '.tsx', '.js', '.json']
+        extensions: ['.ts', '.tsx', '.js', '.json'],
+        alias: {
```

Comment:
Is this because of the `@emotion/cache` package? Otherwise, I think we should prefer changing our imports rather than setting up an alias

Reply:
When testing it in my MBO I needed to have it, now it seems to work without, I guess it was because of @emotion/cache. I removed it.